src/notation/internal/notation.cpp

```diff
@@ -206,6 +206,20 @@ QString Notation::projectWorkTitleAndPartName() const
     return result;
 }
 
+QString Notation::firstTitleText() const
+{
+    if (!m_score) {
+        return QString();
+    }
+
+    engraving::Text* text = m_score->getText(TextStyleType::TITLE);
+    if (!text) {
+        return QString();
+    }
+
+    return text->plainText().replace(u"\n", u"_");
```

Comment:
Some more replacements might be needed, like to avoid / in a filename?

Reply:
Mu 3 did a lot more 'translating' there, if that is the purpose of this method...

Reply:
Thanks! I'll check it out

Reply:
I think this method should not be responsible for replacing characters. IMO, this method should just do what it says, namely return the first title text, in unmodified form. The transformations required to make it usable as a filename should happen elsewhere. 

See also `io::escapeFileName`, although that method is a bit too drastic IMO (it even disallows spaces; for software development, spaces in filenames are not always convenient, but for normal users there's really nothing wrong with it). So perhaps you could create a non-drastic version of `io::escapeFileName`.

Reply:
`createDefaultFileName()` doesn't exist anymore. So I've been trying to use `escapeFileName()` modified to allowed spaces. But that created a new bug which I'm trying to figure out now